framework/console/Application.php

```diff
@@ -180,8 +181,156 @@ public function runAction($route, $params = [])
             $res = parent::runAction($route, $params);
             return is_object($res) ? $res : (int)$res;
         } catch (InvalidRouteException $e) {
-            throw new Exception("Unknown command \"$route\".", 0, $e);
+            $message = "Unknown command \"$route\"";
+            if($alternatives = $this->findAlternatives($route)){
+                $message .= count($alternatives) > 1 ?  "\nDid you mean one of these?" : "\nDid you mean this?";
```

Comment:
In case of only one alternative it could be outputted right away as `Did you mean %s?`